utils/gen_python.jl

```diff
@@ -55,3 +55,30 @@ open(normpath(@__DIR__, "..", "python", "ribasim", "ribasim", "schemas.py"), "w"
     init = Dict("models" => get_models())
     println(io, model_template(; init = init))
 end
+
+function get_connectivity()
+    """
+    Set up a vector contains all possible connecting node for all node types.
+    """
+    [
+        (
+            name = T,
+            connectivity = Set(
+                Ribasim.config.camel_case(x) for x in Ribasim.neighbortypes(T)
+            ),
+            flow_neighbor_bound = Ribasim.n_neighbor_bounds_flow(T),
+            control_neighbor_bound = Ribasim.n_neighbor_bounds_control(T),
+        ) for T in keys(Ribasim.config.nodekinds)
+    ]
```

Comment:
Maybe we want to sort this list? That makes looking through the code a bit simpler?

Reply:
I don't quite understand which list that you meant. But I will mark this down so I can resolve it when you are back!